testing/integration-tests/src/codegen/codegen_tests.rs

```diff
@@ -11,7 +11,7 @@ use subxt_codegen::{CratePath, DerivesRegistry, RuntimeGenerator, TypeSubstitute
 
 fn generate_runtime_interface_from_metadata(metadata: RuntimeMetadataPrefixed) -> String {
     // Generate a runtime interface from the provided metadata.
-    let generator = RuntimeGenerator::new(metadata);
+    let generator = RuntimeGenerator::new(metadata.try_into().unwrap());
```

Comment:
nit: Maybe we can provide an expect message to align with the below `generator`?

Reply:
MAkes sense; I'll add one :)